README.md

```diff
@@ -1,43 +1,135 @@
+<!-- omit in toc -->
 # Cesium Native
 
+[![License](https://img.shields.io/:license-Apache_2.0-blue.svg)](https://github.com/CesiumGS/cesium-native/blob/main/LICENSE)
+[![Build Status](https://github.com/CesiumGS/cesium-native/actions/workflows/build.yml/badge.svg)](https://github.com/CesiumGS/cesium-native/actions/workflows/build.yml)
+[![Release](https://img.shields.io/github/v/tag/CesiumGS/cesium-native?label=release)](https://github.com/CesiumGS/cesium-native/tags)
+
+<table>
+  <tbody>
+    <tr>
+      <td>
+
```

Comment:
Nitpick, but this extra line break is noticably offsetting this table cell :')

![image](https://github.com/user-attachments/assets/21d44174-7765-4a1b-be0d-6fbd13969b53)